packages/cojson-storage-sqlite/src/tests/storage.sqlite.test.ts

```diff
@@ -374,6 +375,8 @@ test("should recover from data loss", async () => {
     [
       "client -> CONTENT Group header: true new: After: 0 New: 3",
       "client -> CONTENT Map header: true new: After: 0 New: 1",
+      "client -> CONTENT Map header: false new: After: 3 New: 1",
+      "storage -> KNOWN CORRECTION Map sessions: header/4",
```

Comment:
This happens because we know push new changes directly instead of diffing.

This test covers data loss, so it's ok to have an extra roundtrip here.